app/models/Airline.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from 'zod';
+
+const AirlineSchema = z.object({
+  id: z.number().optional(),
```

Comment:
Why am i still seeing the id and type field in both swagger and the output?
<img width="1287" alt="image" src="https://github.com/couchbase-examples/nextjs-capella-quickstart/assets/108796209/0afde2e4-68d5-4b6f-88e2-c67026c19664">
Also can you prefill the fields with some default values like we do for other tutorials?

Reply:
id and type are optional only.
![image](https://github.com/couchbase-examples/nextjs-capella-quickstart/assets/44238657/9008627f-9888-4c5d-b1ca-e0d57af473fa)
The user can choose not input those fields

Reply:
Please remove it as they are confusing.
See [requirements](https://docs.google.com/document/d/1_uc_HRKaNSAcEUsUgdAR9EamIotcO4LwOAJZAcMi7fk/edit?pli=1#heading=h.qpuhsn242jgu)

Reply:
![image](https://github.com/couchbase-examples/nextjs-capella-quickstart/assets/44238657/e3478686-d994-41c1-bf57-b74081c177a1)
Filled in with examples